cha/implementaion/playback/playing.tex

```diff
@@ -0,0 +1,50 @@
+\section{Playing \ac{PCM} Data}
+With the data decoded we also create a replacement for the mediaplayer.\mnote[inline]{why btw? could we not simply have used the one we alrdy had?}
```

Comment:
The decoding is part of the default mediaplayer, and therefore it's part of the replacement.